### PR TITLE
Changed the time of effort syncing to hourly at 25th minute

### PR DESCRIPTION
**Pull request by Priyanka-Kaintura**
Change time of effort syncing #2805

### Description
Changed effort syncing time from every four hours to hourly at 25th minute 
To reduce the chance of sending non-updated data to the channel.
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1499,6 +1499,7 @@
                 "merge-source-map": "^1.1.0",
```
Comment (Ayush8923) [inline package-lock.json:1499]: @Priyanka-Kaintura  Please remove this file from this PR. It's not needed.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Left some comment. Please check.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
You don't have to remove the package-lock.json from the codebase. Please revert the file.

**Comment by GauravGusain98** [comment]
> You don't have to remove the package-lock.json from the codebase. Please revert the file.

Only the extra code changes were supposed to be removed

**Comment by Priyanka-Kaintura** [comment]
Can I delete the pull request and the branch? And recreate branch and pull request. Since I have made the mistake and now I’m stuck there. I tried to reset the commit but it still shows the commits already done. If possible, please allow me to delete the branch and create another one.Sent from Mail for Windows Thank-You—

**Review by GauravGusain98 — APPROVED** [review]

**Review by Ayush8923 — APPROVED** [review]

**Pull request merged**